### PR TITLE
ASoC: rt715: avoid reading slave prop from current driver.

### PR DESCRIPTION
**Pull request by jack-cy-yu**
Signed-off-by: Jack Yu <jack.yu@realtek.com>

### CODE REVIEW

File: sound/soc/codecs/rt715-sdw.c [hunk 1]
```diff
@@ -198,8 +198,6 @@ static int rt715_read_prop(struct sdw_slave *slave)
 	unsigned long addr;
 	struct sdw_dpn_prop *dpn;
```
Comment (dbaluta) [inline sound/soc/codecs/rt715-sdw.c:197]: Hi @jack-cy-yu,

Please explain why this commit is needed. Also, please format the commit subject adding some 'tags' to specify the subsystem that is modified by your patch.

Have a look at previous commits git log  sound/soc/codecs/rt715-sdw.c

File: sound/soc/codecs/rt715-sdw.c [hunk 2]
```diff
@@ -7,8 +7,6 @@
  * ALC715 ASoC Codec Driver based Intel Dummy SdW codec driver
```
Comment (dbaluta) [inline sound/soc/codecs/rt715-sdw.c:7]: @jack-cy-yu please split this in two separate commits and for each one explain why the change is needed. Also, make sure you update the 'tags'in commit message according to previous commits in this file.

Should be something like ASoC: codecs: rt715-sdev: ....

File: sound/soc/codecs/rt715-sdw.c (hunk 1 above)
Comment (jack-cy-yu) [reply sound/soc/codecs/rt715-sdw.c:197]: Hi @dbaluta ,

I got a message from @plbossart  as below
"Forgot to mention that I see the following warning for rt715, likely some cleanup necessary as well.
[ 5120.662052] rt715 sdw:3:25d:715:0:1: DP0 node not found!!"

I reviewed the rt715 driver and found this function could be the reason for the warning message, so I removed it from current code just as rt711/rt1308 did.

File: sound/soc/codecs/rt715-sdw.c (hunk 2 above)
Comment (jack-cy-yu) [reply sound/soc/codecs/rt715-sdw.c:7]: @dbaluta aren't these already two separate commits? Or what you mean is two separate PR?

Comment (paulstelian97) [reply sound/soc/codecs/rt715-sdw.c:7]: @jack-cy-yu I think he meant to separate the removal of DEBUG from the change in return value.

**Comment by plbossart** [comment]
@dbaluta @paulstelian97 don't worry too much about the code quality here or format, this is still works in progress. We use PRs to track progress with our friends at Realtek but the patches get squashed/updated quite a bit.
